kong/constants.lua

```diff
@@ -40,6 +40,7 @@ return {
   HEADERS = {
     HOST_OVERRIDE = "X-Host-Override",
     PROXY_LATENCY = "X-Kong-Proxy-Latency",
+    KONG_RESPONSE_LATENCY = "X-Kong-Latency",
```

Comment:
In my opinion, we should either namespace this header (`X-Kong-Admin-Latency`), or group both admin and proxy latencies into `X-Kong-Latency` (the former is probably better, but the later can account for the response streaming back to the client in the proxy case). This is an issue we are already debating internally btw